### PR TITLE
Add Viveka Health

### PR DESCRIPTION
**Pull request by SushilShrestha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Verscend Technologies](http://verscend.com.np/) | Hattisar, Kathmandu | Software research and development center
 - [Vianet Communications Pvt. Ltd.](https://www.vianet.com.np/) | Jawalakhel, Lalitpur | Provider of high speed Internet connectivity and communications solutions for business and individuals
 - [Visualise VR](http://visualisevr.com/) | Pulchowk, Lalitpur |  Help business benefit from mixed reality technologies
+- [Viveka Health](http://www.vivekahealth.com/) | Lalitpur | Outsourcing company (Parent company based in US)
```
Comment (mesaugat) [inline README.md]: @SushilShrestha Where exactly in Lalitpur? Something like `Jawlakhel, Lalitpur` would do.

I am trying to make the list very consistent.

Comment (mesaugat) [reply README.md]: Also, can you remove `(Parent company based in US)` from the description. Every company seems to be based somewhere and I am trying to remove that bias from the description.

Comment (SushilShrestha) [reply README.md]: parent company has been removed but I have to confirm about the exact location. will update the location after further confirmation.

**Comment by SushilShrestha** [comment]
Okay done !!

File: README.md [hunk 2]
```diff
@@ -145,6 +145,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Verscend Technologies](http://verscend.com.np/) | Hattisar, Kathmandu | Software research and development center
 - [Vianet Communications Pvt. Ltd.](https://www.vianet.com.np/) | Jawalakhel, Lalitpur | Provider of high speed Internet connectivity and communications solutions for business and individuals
 - [Visualise VR](http://visualisevr.com/) | Pulchowk, Lalitpur |  Help business benefit from mixed reality technologies
+- [Viveka Health](http://www.vivekahealth.com/) | Sanepa, Lalitpur | Outsourcing company
```
Comment (mesaugat) [inline README.md]: Hello @SushilShrestha, `Outsourcing company` looks a bit generic. Can we add a proper description?

Comment (SushilShrestha) [reply README.md]: Sure I will update those.

**Comment by SushilShrestha** [comment]
Okay, No worries !!

**Pull request merged**